scripts/app.js

```diff
@@ -103,84 +103,60 @@ function InitializeGrid() {
 
     $("#grid").kendoGrid({
         height: 550,
-        autoBind: false,
+        //autoBind: false,
         editable: true,
         sortable: true,
         columns: [
-                    { field: "UnitPrice" },
-                    { field: "ProductName" },
-                    { command: "destroy", width: 150 }
+                    { field: "connections", title: "Connections" },
+                    { field: "touch", title: "Touch" },
+                    { field: "video", title: "Video" }
                 ],
         toolbar: ["create"],
         dataSource: {
             // Handle the push event to display notifications when push updates arrive
             data: products,
             push: function (e) {
-                var notification = $("#notification").data("kendoNotification");
-                notification.success(e.type);
+                //var notification = $("#notification").data("kendoNotification");
+                //notification.success("success");
             },
-            autoSync: true,
+            //autoSync: true,
             schema: {
                 model: {
-                    id: "ProductID",
                     fields: {
-                        "ProductID": { editable: false, nullable: true },
-                        "CreatedAt": { type: "date" },
-                        "UnitPrice": { type: "number" }
+                        "connections": { type: "number" },
+                        "touch": { type: "number" },
+                        "video": { type: "number" }
                     }
-                },
-                data: "data"
+                }
             },
-            sort: [{ field: "CreatedAt", dir: "desc"}],
             transport: {
-                push: function (options) {
-                    console.log(options);
-                    //Listen to the "message" event fired when the server pushes data
-                    //                    ws.addEventListener("message", function (e) {
-                    //                        var result = JSON.parse(e.data);
-
-                    //                        //Check what the push type is and invoke the corresponding callback.
-                    //                        if (result.type == "push-update") {
-                    //                            options.pushUpdate(result);
-                    //                        } else if (result.type == "push-destroy") {
-                    //                            options.pushDestroy(result);
-                    //                        } else if (result.type == "push-create") {
-                    //                            options.pushCreate(result);
-                    //                        }
-                    //                    });
-
+                push: function (callbacks) {
+                    console.log(callbacks);
+                    consumer.on("create", function(result) {
+                      console.log("push create");
```

Comment:
this should be called when you want to add new item to the grid ( in case your server datasource has a new item)